<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filterForm</class>
 <widget class="QWidget" name="filterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">filterForm
{
     background-color: rgb(12,53,64);
}</string>
  </property>
  <widget class="QFrame" name="filterMainFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>230</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="conditionLineEdit">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>50</y>
      <width>64</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Текст</string>
    </property>
   </widget>
   <widget class="QComboBox" name="tableComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>210</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <item>
     <property name="text">
      <string>Оберіть колонку</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>dad</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>asdasd</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>asd</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>asd</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>asdas</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="conditionComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>Оберіть умову</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&lt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&gt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&lt;=</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&gt;=</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>=</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>like</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>between</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="filterPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Фільтрувати</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="conditionLineEdit_2">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Текст</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearFilterPushButton">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>80</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recources.qrc">
      <normaloff>:/img/blueMenuIcon/clearFilterIco.png</normaloff>:/img/blueMenuIcon/clearFilterIco.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QDateEdit" name="conditionDataEdit">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximumDate">
     <date>
      <year>2099</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>2000</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="recources.qrc"/>
 </resources>
 <connections/>
</ui>
